test/src/main/java/org/openjdk/skara/test/TestHostedRepository.java

```diff
@@ -229,7 +229,7 @@ public Optional<String> fileContents(String filename, String ref) {
     }
 
     @Override
-    public void writeFileContents(String filename, String content, Branch branch, String message, String authorName, String authorEmail) {
+    public void writeFileContents(String filename, String content, Branch branch, String message, String authorName, String authorEmail, boolean createNewFile) {
```

Comment:
This implementation should also enforce the new behavior so we catch any mistakes in testing.

Reply:
Will fix it